### ISSUE TITLE
Processing 4 and Python on MacOs don't work out of the box

### ISSUE THREAD

**Issue by Balky79** [jdf/processing.py#414]
I don't use iCloud and it was full, so processing could not store it's defaults there. 

I've changed that in settings to go to my local folder, killed/restarted processing. 

Installed python support, tried to run an example:

It "starts" the example (UI looks like it started) but nothing happens in console. Once I stop execution, console displays:

Hmm. Can't load native code to bring window to front using the absolute path: /Users/igors/Docs/Processing/modes/PythonMode/mode/libjniosx.dylib.
Hmm. Can't load native code to bring window to front.

File is on the path, I can see it when I go to it manually. 

Do I need to symlink something, or give some permissions somewhere? 
Anyone else experiencing the same?

P5.js and Java work fine. 

M1 Silicon Mac
Ventura 13.13.1

**Comment by KopyWasTaken** [comment]
Yeah I'm also having this issue. Have you found a way to fix it yet?
